fuel-metrics/src/txpool_metrics.rs

```diff
@@ -0,0 +1,59 @@
+use lazy_static::lazy_static;
+pub use prometheus_client::metrics::histogram::Histogram;
+use prometheus_client::registry::Registry;
+use std::{
+    boxed::Box,
+    default::Default,
+};
+
+pub struct TxPoolMetrics {
```

Comment:
Could you add a description for each field? Maybe copy/paste from the `init` function or add a comment like "For more details, check the `init` function." (the same for all metrics)